include/range/v3/action/insert.hpp

```diff
@@ -76,23 +76,76 @@ namespace ranges
                 return unwrap_reference(cont).insert(p, n, std::forward<T>(t));
             }
 
+            namespace impl {
```

Comment:
In range-v3, the detail namespace is called `detail` everywhere. Also, to avoid cluttering the Doxygen output with details, I wrap details with `/// \cond` and `/// \endcond`.